contrib/seeds/README.md

```diff
@@ -8,21 +8,11 @@ and remove old versions as necessary (at a minimum when GetDesirableServiceFlags
 changes its default return value, as those are the services which seeds are added
 to addrman with).
 
-The seeds compiled into the release are created from sipa's DNS seed data, like this:
+The seeds compiled into the release are created from sipa's DNS seed and AS map
+data, like this:
```

Comment:
nit, maybe be explicit regarding the directory to run these commands from
```suggestion
data. Run the following commands from the `/contrib/seeds` directory:
```

Reply:
Makes sense.